python/pyspark/sql/functions.py

```diff
@@ -1729,6 +1729,29 @@ def from_json(col, schema, options={}):
     return Column(jc)
 
 
+@ignore_unicode_prefix
+@since(2.1)
+def to_json(col, options={}):
+    """
+    Converts a column containing a [[StructType]] into a JSON string. Returns `null`,
+    in the case of an unsupported type.
+
+    :param col: struct column
```

Comment:
Would    `:param col: name of column containing the struct` maybe be more consistent with the other pydocs for the functions? (I only skimmed a few though so if its the other way around thats cool).